tutorials/llm/llama-3/README.rst

```diff
@@ -0,0 +1,207 @@
+Llama 3 LoRA fine-tuning and deployment with NeMo Framework and NVIDIA NIM
+==========================================================================
+
+`Llama 3 <https://blogs.nvidia.com/blog/meta-llama3-inference-acceleration/>`_
+is an open source large language model by Meta that delivers
+state-of-the-art performance on popular industry benchmarks. It has been
+pretrained on over 15 trillion tokens, and supports an 8K token context length.
+It is available in two sizes: 8B and 70B, and it has two variants for
+each size - base pretrained and instruction tuned.
+
+`Low-Rank Adaptation (LoRA) <https://arxiv.org/pdf/2106.09685>`__ has
+emerged as a popular Parameter Efficient Fine-Tuning (PEFT) technique
+that tunes a very small number of additional parameters as compared to
+full fine-tuning, thereby reducing the compute required.
+
+`NVIDIA NeMo
+Framework <https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html>`__
+provides tools to perform LoRA on Llama 3 to fit your usecase, which can
+then be deployed using `NVIDIA NIM <https://www.nvidia.com/en-us/ai/>`__
+for optimized inference on NVIDIA GPUs.
+
+.. figure:: ./img/e2e-lora-train-and-deploy.png
+  :width: 1000
+  :alt: Diagram showing the steps for customizing NVIDIA NIM with LoRA using NVIDIA NeMo. The steps include converting models to .nemo format, creating LoRA adapters with NeMo, and then using the LoRA adapter with NIM for inference on the customized model.
```

Comment:
Steps for LoRA customization using the NVIDIA NeMo Framework and deployment with NVIDIA NIM